lib/timezone/timezone_local.ex

```diff
@@ -44,13 +44,18 @@ defmodule Timex.Timezone.Local do
 
   def lookup(), do: Date.now |> lookup
   def lookup(%DateTime{} = date) do
-    case :os.type() do
-      {:unix, :darwin} -> localtz(:osx, date)
-      {:unix, _}       -> localtz(:unix, date)
-      {:nt}            -> localtz(:win, date)
-      {:win32, :nt}    -> localtz(:win, date)
-      _                -> raise "Unsupported operating system!"
+    tz = Application.get_env(:timex, :timezone)
```

Comment:
I think we should change it from `:timezone` to `:local_timezone` just to be specific as to what is being set here.